modules/git/url/url.go

```diff
@@ -0,0 +1,71 @@
+// Copyright 2022 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package url
+
+import (
+	"fmt"
+	"net/url"
+	"regexp"
+	"strings"
+)
+
+// URL represents a git remote URL
+type URL struct {
+	*url.URL
```

Comment:
It's ambiguous to read `url.URL` in code. It looks like the Golang built-in `url.URL`

Should we call it `giturl.GitURL`?

Reply:
I don't think it's necessary. A similar form makes users know it quickly.